.nuspec/Xamarin.Forms.Android.targets

```diff
@@ -2,4 +2,12 @@
   <ItemGroup>
     <ProguardConfiguration Include="$(MSBuildThisFileDirectory)proguard.cfg" />
   </ItemGroup>
+  <Target Name="ValidateTargetFrameworkVersionForForms" BeforeTargets="CoreCompile">
```

Comment:
needs the condition so we can disable the validation (see other checks in the same file)